index.html

```diff
@@ -617,17 +617,24 @@ <h2>Queue a <code>PerformanceEntry</code></h2>
         </li>
         <li>For each <var>observer</var> in <var>interested observers</var>:
           <ol>
-            <li>Append <var>new entry</var> to <var>observer</var>'s
+            <li>Append <var>newEntry</var> to <var>observer</var>'s
             <a>observer buffer</a>.
             </li>
           </ol>
         </li>
+        <li>Let <var>relevantGlobal</var> be <var>newEntry</var>'s <a>
+          relevant global object</a>.
+        </li>
+        <li>
+          Let <var>tuple</var> be the <a>relevant performance
+          entry tuple</a> of <var>relevantGlobal</var> and <var>entryType</var>.
```

Comment:
Those should probably come in the opposite order, since that's the order the "relevant performance entry tuple" inputs are defined to come in.